app/policies/external_activity_policy.rb

```diff
@@ -30,7 +30,7 @@ def set_private_before_matedit?
   end
 
   def copy?
-    user.present?
+    not_anonymous?
```

Comment:
Perhaps this isn't consistent but, generally we pass `current_user` to the policies not `current_visitor`. `current_user` will be nil if the user is anonymous.   I believe this is the main place this is done: https://github.com/concord-consortium/rigse/blob/master/app/controllers/application_controller.rb#L30